Travel_fellowships.md

```diff
@@ -73,6 +73,7 @@ The OBF Event Fellowships are available for the awardees to cover conference reg
 
 The fellowship can be used to cover conference registration fees and direct travel costs (such as airfare, train, metro, bus and taxi), hotel, poster printing (if presenting a poster at the conference), and/or child-care up to a value of USD 1,000. 
 Note that rental car expenses and personal car mileage are not eligible for reimbursement. 
+Hotel accommodation earlier than the night before the first day of the event, or beyond the night the event ends, requires exceptional justification.
```

Comment:
I guess what _exceptional_ means in practice is in the eye of the beholder, i.e., ambiguous? Also, didn't we want to add that approval is at the Board's discretion? Stating both seems at least partly redundant, so I would perhaps simplify to the following:

Reply:
```suggestion
Lodging for earlier than the night before the first day of the event, or beyond the night the event ends, requires justification and is subject to Board approval.
```

Reply:
BTW I just notice that we use explicitly _hotel_ above in the enumeration in line 74 (2 lines before this change). Perhaps worth generalizing this to _accommodation_ or _lodging_?

Reply:
Yes, hotels is a little too narrow isn't it.